searx/search/processors/online.py

```diff
@@ -139,15 +148,15 @@ def search(self, query, params, result_container, start_time, timeout_limit):
             self.handle_exception(result_container, e, suspend=True)
             logger.error("engine {0} : HTTP requests timeout"
                          "(search duration : {1} s, timeout: {2} s) : {3}"
-                         .format(self.engine_name, time() - start_time,
+                         .format(self.engine_name, default_timer() - start_time,
```

Comment:
`start_time` is set using `default_timer()` : https://github.com/searxng/searxng/blob/d36adfa59f242a8775ad74245c696d62b7727a36/searx/search/__init__.py#L188